src/components/common/Inputs/input.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
```

Comment:
Also, this was changed to TextInput. The best practice is to avoid names of components that are similar to usual HTML tags.